packages/theme/src/cli/commands/theme/pull.ts

```diff
@@ -61,58 +54,27 @@ If no theme is specified, then you're prompted to select the theme to pull from
       description: 'Proceed without confirmation, if current directory does not seem to be theme directory.',
       env: 'SHOPIFY_FLAG_FORCE',
     }),
-    legacy: Flags.boolean({
-      hidden: true,
-      description: 'Use the legacy Ruby implementation for the `shopify theme pull` command.',
-      env: 'SHOPIFY_FLAG_LEGACY',
-    }),
   }
 
   static cli2Flags = ['theme', 'development', 'live', 'nodelete', 'only', 'ignore', 'force', 'development-theme-id']
 
   async run(): Promise<void> {
-    showEmbeddedCLIWarning()
-
     const {flags} = await this.parse(Pull)
-    const store = ensureThemeStore(flags)
-    const adminSession = await ensureAuthenticatedThemes(store, flags.password)
-
-    const developmentThemeManager = new DevelopmentThemeManager(adminSession)
-    const developmentTheme = await (flags.development
-      ? developmentThemeManager.find()
-      : developmentThemeManager.fetch())
-
-    if (!flags.legacy) {
-      const {path, nodelete, live, development, only, ignore, force} = flags
-
-      const theme = await findOrSelectTheme(adminSession, {
-        header: 'Select a theme to open',
-        filter: {
-          live,
-          theme: development ? `${developmentTheme?.id}` : flags.theme,
-        },
-      })
 
-      await pull(theme, adminSession, {path, nodelete, only, ignore, force})
-
-      return
+    const pullFlags: PullFlags = {
+      path: flags.path,
```

Comment:
What do you think about adding the documentation in the properties here instead?

Something like this:
```ts
/**
 * The directory path to download the theme.
 */
path?: string
```

The benefit I notice is that users get docs for their properties:
<img width="90%" src="https://github.com/user-attachments/assets/a2b9200c-52f2-48cd-92b4-f1853e76d2d5">

And the documentation of functions gets a bit easier to digest:
<img width="90%" src="https://github.com/user-attachments/assets/8b14602e-b19f-4f1d-8608-630fd5cd1573">

Reply:
I like this idea!